### PR TITLE
Add JetStream parameters to openfaas chart README

### PR DESCRIPTION
**Pull request by welteki**
Signed-off-by: Han Verstraete (OpenFaaS Ltd) <han@openfaas.com>

## Description

Add the JetStream configuration parameters to the openfaas chart README

## Motivation and Context

Keep documentation up to date after merging #987 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: chart/openfaas/README.md [hunk 1]
```diff
@@ -519,6 +525,7 @@ See [values.yaml](./values.yaml) for detailed configuration.
 | `nats.metrics.enabled` | Export Prometheus metrics for NATS, no multi-arch support  | `false` |
 | `nats.metrics.image` | Container image used for the NATS Prometheus exporter | See [values.yaml](./values.yaml) |
 | `nats.resources` | Resource limits and requests for the nats pods | See [values.yaml](./values.yaml) |
+| `nats.streamReplication` | JetStream stream replication factor | `1` |
```
[the remark this answers is not in the document]
Comment (alexellis) [reply chart/openfaas/README.md:528]: Perhaps also set it here? https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/values-pro.yaml#L47

[the remark this answers is not in the document]
Comment (alexellis) [reply chart/openfaas/README.md:528]: Can you raise an issue for our checker about streamReplication, and making sure it's 3 or higher for any jetstream users?

[the remark this answers is not in the document]
Comment (welteki) [reply chart/openfaas/README.md]: I updated the description to recommend 3 in production and also added it the OpenFaaS Pro values file.

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**